certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -423,3364 +491,12591 @@
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    },
+    "/api/v1/Events/Registrants/{recordNumber}/Attended": {
+      "put": {
+        "tags": [
+          "Events"
+        ],
+        "summary": "Mark Registrant as Attended",
+        "description": "Mark the Registrant as Attended for the specified Events.",
+        "operationId": "Mark-Registrant-Attended",
+        "parameters": [
+          {
+            "name": "recordNumber",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Record Number.",
+            "x-ms-summary": "Record Number",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "x-ms-summary": "Data List",
+            "required": true,
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "eventOrSessionCode": {
+                    "type": "string",
+                    "description": "Event or Session Code.",
+                    "title": "Code"
+                  }
+                },
+                "required": [
+                  "eventOrSessionCode"
+                ]
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          }
+        }
+      }
+    },
+    "/api/v1/Awards/{id}/Nominations": {
+      "post": {
+        "tags": [
+          "Awards"
+        ],
+        "summary": "Add Award Nomination",
+        "description": "Add a new Award Nomination.",
+        "operationId": "Awards-AddAwardNomination",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Award Id.",
+            "x-ms-summary": "Award Id",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "awardNominationData",
+            "in": "body",
+            "description": "",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AwardNominationData"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {}
```

Comment:
@RjImpx There are many empty response scehma in the swagger.  Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Reply:
@sriyen-msft Thank you. we updated the PR to make sure that there are no empty responses.We removed the schema property for those that do not have a response.